assets/js/data/cart/index.ts

```diff
@@ -27,6 +28,10 @@ const registeredStore = registerStore< State >( STORE_KEY, {
 } );
 
 registeredStore.subscribe( pushChanges );
+registeredStore.subscribe( async () => {
```

Comment:
Added https://github.com/woocommerce/woocommerce-blocks/issues/6630 to see if this can be improved